pom.xml

```diff
@@ -585,6 +585,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/x
             <exclude>licenses-binary/**</exclude>
             <exclude>dev-support/docker/pkg-resolver/packages.json</exclude>
             <exclude>dev-support/docker/pkg-resolver/platforms.json</exclude>
+            <exclude>dev-support/docker/vcpkg.json</exclude>
```

Comment:
this path is wrong. missing /vcpkg after /docker
should be
```
dev-support/docker/vcpkg/vcpkg.json
```

Reply:
Thanks for the catch @ayushktn.

Reply:
Thanks, fixed.